src/Raven.Server/Commercial/LetsEncrypt/LetsEncryptCertificateUtil.cs

```diff
@@ -40,7 +40,7 @@ internal static (byte[] CertBytes, CertificateDefinition CertificateDefinition,
 
     public static async Task WriteCertificateAsPemToZipArchiveAsync(string name, byte[] rawBytes, string exportPassword, ZipArchive archive)
     {
-        var a = new Pkcs12StoreBuilder().Build();
+        var a = new Pkcs12StoreBuilder().SetEnableOracleTrustedKeyUsage(false).Build();
```

Comment:
are we using SetEnableOracleTrustedKeyUsage every time we do Pkcs12StoreBuilder()? if so, maybe we need a wrapper method?